R/propre_brochure.R

```diff
@@ -0,0 +1,71 @@
+#' gouvdown PDF brochure - PROPRE style
+#'
+#' @inheritParams html_gouv
+#' @param ... Additional arguments passed to \code{bookdown::\link{html_document2}()}.
+#' @param css A character vector of additionnal CSS file paths.
+#' @param width_main_column Width of the main text column (in %)
+#'
```

Comment:
```suggestion
#' @param made_with Character. Add extra information to show how the document was made.
#'
```

Je pense qu'il manque cette documentation de fonction et le passage de `roxygen2::roxygenise()` pour que ce soit fonctionnel